opentech/settings/base.py

```diff
@@ -172,6 +173,9 @@
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 MEDIA_URL = '/media/'
 
+AUTH_USER_MODEL = 'users.User'
```

Comment:
This should point to `opentech.User`

Reply:
it is pointing to `app_label.ModelName` as required

Reply:
ah, my mistake in typing. I meant to say: `opentech.users.User`, `opentech.users` is what is in installed apps as the app_label. 

Does the above auth model definitely point at the custom auth model table?

Reply:
:nod: